capstone-backend/src/main/java/com/google/sps/servlets/ClubServlet.java

```diff
@@ -90,7 +90,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       clubs.add(club);
     }
     Gson gson = new Gson();
-    response.setContentType("application/json;");
+    response.setContentType("text/json;");
```

Comment:
I think I may have told you wrongly when I said to use text/json instead of application/json. It looks like application/json is the official MIME media type. I think they have the same functionality though!
https://stackoverflow.com/a/477819

Reply:
Oh interesting, I'll change it back.